source/13_envs/acrobot_zh.rst

```diff
@@ -0,0 +1,160 @@
+Acrobot
+~~~~~~~~~~~~~~~~~~
+
+概述
+=======
+Acrobot机器人系统是强化学习中的经典控制问题。该系统包括两个关节和两个连杆，其中两个连杆之间的关节可以被驱动。系统的初始状态是向下悬挂的。目标是在驱动关节上施加力矩，使连杆的自由端摆动到指定高度。如下图所示。
+
+.. image:: ./images/acrobot.gif
+   :align: center
+   :scale: 80%
+安装
+====
+
+安装方法
+--------
+
+Acrobot 环境内置在 gym 中，直接安装 gym 即可。其环境 id 是\ ``Acrobot-v1`` \。
+
+.. code:: shell
+
+    pip install gym
+    
+验证安装
+--------
+
+在 Python 命令行中运行如下命令验证安装成功。
+
+.. code:: shell 
+
+    import gym
+    env = gym.make('Acrobot-v1')
+    obs = env.reset()
+    print(obs)  
+
+环境介绍
+=========
+
+动作空间
+----------
+
+Acrobot 的动作空间属于离散动作空间，有 3 个离散动作，分别是施加+1力矩, -1力矩和不施加力矩。
+
+-  \ ``-1的力矩`` \: 0 表示向关节施加-1的力矩。
+
+-  \ ``0的力矩`` \: 1 表示向关节施加0的力矩。
+
+-  \ ``+1的力矩`` \: 2 表示向关节施加+1的力矩。
+
+使用 gym 环境空间定义则可表示为：
+
+.. code:: python
+    
+    action_space = spaces.Discrete(3)
```

Comment:
you should import `spaces` first or use `gym.spaces`

Reply:
fixed